### PR TITLE
docs(alert):  use secondary rh-buttons to visually reflect importance

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1.  Updated demos to use `rh-button variant="secondary"` to visually reflect button importance.
2. Updated Alert docs to reference latest images and content.

## Testing Instructions

1. View [Deploy Preview demos](https://deploy-preview-1857--red-hat-design-system.netlify.app/elements/alert/demos/)

### CODE REVIEW

File: elements/rh-alert/docs/00-overview.md [hunk 1]
```diff
@@ -2,14 +2,21 @@
 
 {{ tagName | getElementDescription }}
 
-{% repoStatusList repoStatus=repoStatus %}
+<uxdot-example width-adjustment="538px">
+  <img src="{{ 'alert-sample.svg' | url }}" alt="Two examples of the alert element" width="538px">
```
Comment (bennypowers) [inline elements/rh-alert/docs/00-overview.md:6]:
```suggestion
  <img src="alert-sample.svg" alt="Two examples of the alert element" width="538px">
```

**Review by bennypowers — COMMENTED** [review]
Does this need guidelines prose?

**Comment by coreyvickery** [comment]
> Does this need guidelines prose?

@bennypowers I have updated docs ready in Figma once this gets figured out.

**Comment by zeroedin** [comment]
> > Does this need guidelines prose?
> 
> @bennypowers I have updated docs ready in Figma once this gets figured out.

Taking a look at these doc updates now

**Comment by coreyvickery** [comment]
> Taking a look at these doc updates now
@zeroedin Let me know if you need the Figma link.

**Comment by zeroedin** [comment]
> @zeroedin Let me know if you need the Figma link.

@coreyvickery docs all updated, PTAL.

**Comment by coreyvickery** [comment]
@zeroedin Some tiny edits.

### Style page
- Under `Inline`, it looks like there's an extra `\` in the image container
- Change the `Spacing` heading to `Space` (my fault if that was in the Figma, I've corrected it)
- Under `Spacing`, the second image has a double border or something

### Guidelines page
- Under `Small screens`, image looks a little big

### Accessibility page
- From Greg, under `Keyboard interactions`, update the content in the table (below)
- Add code styling to content in `Key` column

| Key | Result |
| - | - |
| `Tab` | Moves focus to the next interactive element (e.g., from the close button to the first action button in the alert) |
| `Shift+Tab` | Moves focus to the previous interactive element (e.g., from the first action button in the alert back to the close button) |
| `Enter` | No change to content. |

Great job, looks awesome.

**Review by bennypowers — APPROVED** [review]

**Pull request merged**